### ISSUE TITLE
A11y_SQL Azure Data Studio_Extensions -uninstall_Screen reader: After invoking "uninstall" button Screen Reader is not announcing "Uninstallation" status whether it is uninstallation is completed or not.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24629]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG4.1.3; #ScreenReader; #Win32; #A11yWCAG2.1; #FTP; #STP; #Status Messages; #NVDA; #Narrator;

### Environment Details:
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11
Screen Reader: NVDA Version 2023.1(2023.1.0.27913)

### Repro Steps:

1. Launch Azure Data Studio.
2. From the left menu, Tab till the 'Extensions' icon and press enter to open the Extension Manager.
3. Choose any extension from the library.
4. Now Turn on NVDA/Narrator.
5. Tab till 'uninstall' button to uninstall and press enter.
6. Now verify that NVDA/Narrator is not announcing uninstallation is completed or not.

### Actual Results:
After invoking "uninstallation" button Screen Reader is not announcing "uninstallation" status whether it is uninstallation is completed or not.

**Note:**
**1.Similar issue observed with NVDA**

### Expected Results:
After invoking "uninstallation" button Screen Reader should announce "uninstallation" status whether it is installation is completed or not.

### User Impact:
Screen reader user will not be able to know uninstallation status whether it is uninstallation is completed or not. If NVDA/Narrator not announcing.

### Attachment:

[Uninstalled not fixed Narrator.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/ef2fc860-3a79-4c9b-b48a-8ff0f341a8e7)

[Uninstalled not fixed NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/0d875dfb-d55f-442f-8141-4a9cc847c6d7)

**Comment by kisantia** [comment]
@PadminiGariki 
After the uninstallation is started, the screenreader says "Please reload Azure Data Studio to complete the uninstallation of the extension" - is this not acceptable as announcing that ADS needs to be restarted to complete the uninstallation?
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/9dde256b-6d51-4638-a8e0-a1b83e85d135)

**Comment by PadminiGariki** [comment]
verified the bug in this environment [Latest insiders build](https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release).
As per above comment  we check this bug. "Please reload Azure Data Studio to complete the uninstallation of the extension". This message also acceptable. Because screen reader know about the uninstallation is completed or not.
Here Issue is fixed. 
Hence that Bug is Closed.
Attachments is added for reference.
#Closed; #Regressed:10-31-23;

[Issue is fixed with Narrator.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/21b647b4-0a07-4693-84de-77245836a42a)

[Issue is fixed with NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/17e1d8e2-a2ae-453b-a8c8-49b59be43e3f)

**Comment by kisantia** [comment]
closing issue after the confirmation above.
